train-pytorch.py

```diff
@@ -235,7 +235,6 @@ def run():
                         ],
                         path=args.filelog,
                         file_prefix=short_name,
-                        args=args
```

Comment:
I wanted this so people can customize the file logger and pass argument to it.

Reply:
Makes sense, however the constructor of `FileLogger` doesn't accept an args (or *args) keyword.